08_Intro_jQuery/solution_student/select_fun/js/main.js

```diff
@@ -7,21 +7,29 @@
 // 1. Go to style.css and add a class called fun. Style that class with your choice of background-color and text color.
 
 // 2. Using jQuery, add the fun class to the elements with the class .box
-
+$('.box').addClass('.fun');
 
 // 3. When #box2 is clicked on
 	// a) Remove the crazy class from all the items with a class of .box
 	// b) Add the crazy class to the h1
 	// c) Slide toggle #box3
-
+$('#box2').on('click', function() {
+	$('.box').removeClass('.crazy');
+	$('h1').addClass('crazy');
+	$('#box3').slideToggle();
+});
 
 // 4. When any .box is clicked, change the text in the header's h1 tag to read "jQuery Ninja"
-
-
+/**$('.box')on('click', function() {
```

Comment:
You were so close on this one! You just forgot the period before the word 'on'! You wrote:
```
$('.box')on('click', function() {
```
but should have written:
```
$('.box').on('click', function() {
```
That's all!